src/Skeleton/skeleton.module.scss

```diff
@@ -0,0 +1,49 @@
+@import '../variables.scss';
+
+.pulse {
+  animation: pulse 1.5s ease-in-out 0.5s infinite;
+  display: block;
+  background-color: rgba(0, 0, 0, 0.11);
+  margin-bottom: 15px;
+}
+
+.transform {
+  transform-origin: 0 55%;
+  transform: scale(1, 0.8);
+}
+
+.text {
+  height: 20px;
+  border-radius: 4px;
+}
+
+.image {
+  height: 110px;
+}
+
+.button {
+  border-radius: 25px;
+  height: 44px;
+  width: 100%;
+}
+
+.button-secondary {
+  background: transparent;
+  border: 1px solid rgba(0, 0, 0, 0.11);
```

Comment:
can we figure out what color on variables.scss this maps to if any? (closest possible would work too imo)

Reply:
Closest one I found is $gray-3! Good catch on this.